### DISCUSSION TITLE
Cafe error

### DISCUSSION THREAD

**Discussion by Ram0619** [hahnlab/CAFE5#201] [Q&A] [answered]
Hi, I'm basically a biologist and I am getting into the field of bioinformatics. I'm trying to run CAFE analysis to understand the evolution of a wild legume species against cultivated species and other close relatives.  I used orthofinder to determine the orthogroups and species tree. I used the following script,

#! cafe

# version
# date

load -i Orthogroups_input
[Orthogroups_input.csv](https://github.com/user-attachments/files/16837094/Orthogroups_input.csv)
.tsv -t 10 -l logfile.txt

tree (sp8:10,((sp5:2.91386,sp3:2.91386):2.04214,((sp1:1.11798,sp2:1.11798):2.26292,(sp4:2.82388,(sp7:1.6567,sp6:1.6567):1.16718):0.557025):1.57509):5.04401)

lambda -s -t (1,((1,1)1,((2,2)2,(2,(2,2)2)1)1)1)

errormodel -model error.txt -p 0.01

report report.txt

I'm getting a warning message as follows,
WARNING: Calculated posterior probability for family OG0000000 = 0
Lambda : 0.03280017385259,0.08005850450136 & Score: -inf
.WARNING: Calculated posterior probability for family OG0000000 = 0
Lambda : 0.03444018254522,0.08005850450136 & Score: -inf
.WARNING: Calculated posterior probability for family OG0000000 = 0
Lambda : 0.03280017385259,0.48035102700815 & Score: -inf. Additionally, I'm attaching the files associated with the analysis
 Can someone help me out!!
Regards,
Ram

**Comment by hahnlab-user** [comment, marked as answer]
Hi, your input data have extremely large variances in family size, which are causing problems for CAFE. (They also seem quite weird to me.) This is the problem--no likelihoods can be calculated.

Matt
